fem/pfespace.hpp

```diff
@@ -190,15 +190,15 @@ class ParFiniteElementSpace : public FiniteElementSpace
    /// Updates the internal mesh pointer. @warning @a new_mesh must be
    /// <b>topologically identical</b> to the existing mesh. Used if the address
    /// of the Mesh object has changed, e.g. in @a Mesh::Swap.
-   virtual void UpdateMeshPointer(Mesh *new_mesh);
+   void UpdateMeshPointer(Mesh *new_mesh) override;
```

Comment:
Why is `virtual` removed from all these functions?

Reply:
These functions are all overrides of parent virtual functions, for which `override` is the appropriate keyword. Marking a function in a child class as `override` ensures that there exists a parent function to displace, whereas marking it `virtual` only can cause errors if the parent class method is deleted, as such a virtual function is still valid. Secondary to this I changed them because once you mark one method as override in a class, compiler warnings highlight all the rest if they should be override instead.